src/stories/documentation.mdx

```diff
@@ -0,0 +1,11 @@
+## GreenwoodJS Website Documentation
+
+
+We use a combination of technologies to build the GreenwoodJS website
```

Comment:
Looks like the formatting is a bit off here and there aren't any links, so let's make sure to get those in
![Screenshot 2024-04-13 at 6 19 26 PM](https://github.com/ProjectEvergreen/www.greenwoodjs.dev/assets/895923/e324624d-08aa-4b90-9c31-275e5c564a6e)

You can check the source from the linked reference for an example
https://raw.githubusercontent.com/AnalogStudiosRI/www.blissfestri.com/main/src/stories/Documentation.stories.mdx

Also, I think if we can have these side links uppercased (I think we just need to rename the files to _Documentation_ etc, that would be ideal.